docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.1
+    environment:
+      CALYPSO_ENV: 'development'
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules:/calypso/node_modules
```

Comment:
I believe that this can fail because it lacks a prefix.

if you use `PWD` I think that will turn these into absolute paths and prevent certain problems

also, is not the second volume for `node_modules` redundant since it will be brought in on line 10?

```
volumes:
  - $PWD:/calypso
```